File: dayFour/Python/list_addition_target.py
def addition(number_list, target_number):
    new_list = []
    for number in range(len(number_list)):
    
            for index in range(number+1,len(number_list)):
            
                if number_list[number] + number_list[index] == target_number:
                
                    new_list.append(number_list[number]) 
                    new_list.append(number_list[index])
                    break
            
    return new_list

File: dayFour/JavaScript/listAdditionTarget.js
function addition(numberList, targetNumber){
    let newList = []
    for(number = 0; number < numberList.length; number++){
    
        for(index = number+1; index < numberList.length; index++){
            
            if (numberList[number] + numberList[index] == targetNumber){
                
                newList[0] = numberList[number];
                newList[1] = numberList[index];
                break
            }
                
        }
     }     
    return newList
}

module.exports = {addition}


